### PR TITLE
Fix several tests; removed unused list in the navbar

### PR DESCRIPTION
**Pull request by rachelpatterson86**
Notes:
-Added `chai-spy` package to enable spying on an object during testing.
-Fix several tests

Other:
- 3 tests are still failing and will need to spend more time to figure out why this is happening.
- To check the test, `required ui-grid` in OrganizationDetails.js needs to commented out. Will need to make this work with tests in the future.

### CODE REVIEW

**Comment by EricWehrly** [comment]
Rachel, can you put this in a branch off of the main project so that circleci can build it?

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "bootstrap-sass": "^3.3.7",
     "browser-sync": "^2.11.1",
     "chai": "^3.4.0",
+    "chai-spies": "git+https://github.com/chaijs/chai-spies.git",
```
Comment (EricWehrly) [inline package.json]: Is chai spies not in npm?

File: client/app/common/navbar/navbar.html [hunk 2]
```diff
@@ -11,13 +11,13 @@
             </li>
           </ul>
         </div>
-        <ul class="nav navbar-nav" ng-if="$ctrl.auth.getAuthState()">
+        <!-- <ul class="nav navbar-nav" ng-if="$ctrl.auth.getAuthState() && UsersCrtl.subTitle">
```
Comment (EricWehrly) [inline client/app/common/navbar/navbar.html]: Did the subTitle check not work in the controller? We should either remove this if we're not going to use it, or structure the if in such a way that lets us enable it when modules call for it.

File: package.json (hunk 1 above)
Comment (rachelpatterson86) [inline package.json]: Nope. And it needs to be this version.

File: client/app/common/navbar/navbar.html (hunk 2 above)
Comment (rachelpatterson86) [inline client/app/common/navbar/navbar.html]: subTitle is not working. It's in the old/ directory though. Should we add subTitle logic? If so, I can rework and just open up another PR for this aspect.

**Pull request closed without merging**